.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.6
+  - 2.1.10
```

Comment:
Yes, I already use them for rubies 2.1, 2.2 and 2.3. If it's ok to use them to test against two ruby versions simultaneously, I can do it, although I'm not sure it won't create any overlaps. I've taken these from your PR to have the latest Stripe API version for testing.